src/styles/standup.module.scss

```diff
@@ -0,0 +1,180 @@
+$theme-CornFlowerBlue: #1c64f2;
+$theme-Biscay: #1c325e;
+$theme-white: #ffffff;
+$theme-FrenchRose: #f1416c;
+$theme-FunBlue: #1e429f;
+$theme-CeriseRed: #d61f69;
+$theme-PaleSky: #6b7280;
+$theme-Mischka: #d1d5db;
+$theme-AthensGray: #e5e7eb;
+$radius: 8px;
+
+.standupContainer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 30px;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+}
+
+.standupBanner {
+    background-color: $theme-Biscay;
+    width: 83.6875%;
```

Comment:
what is this percentage? use the guidelines do not copy the styles from styles tab of figma

Reply:
fixed